packages/integrations/sitemap/src/index.ts

```diff
@@ -16,6 +16,7 @@ type SitemapOptions =
 			 * ```
 			 */
 			filter?(page: string): string;
+			customPages?: Array<string>;
```

Comment:
Would you mind adding a JSDoc style comment to document this option and include simple example usage? That would be very helpful!